docs/website/docs/building-from-source/riscv.md

```diff
@@ -0,0 +1,125 @@
+# RISC-V cross-compilation
+
+Running on a platform like RISC-V involves cross-compiling from a _host_
+platform (e.g. Linux) to a _target_ platform (a specific RISC-V CPU architecture
+and operating system):
+
+* IREE's _compiler_ is built on the host and is used there to generate modules
+  for the target
+* IREE's _runtime_ is built on the host for the target. The runtime is then
+  pushed to the target to run natively.
+
+## Prerequisites
+
+### Host environment setup
+
+You should already be able to build IREE from source on your host platform.
+Please make sure you have followed the [getting started](./getting-started.md)
+steps.
+
+### Install RISC-V cross-compile toolchain and emulator
+
+You'll need a RISC-V LLVM compilation toolchain and a RISC-V enabled QEMU
+emulator.
+
+See instructions in the following links
+
+* [Clang getting started](https://clang.llvm.org/get_started.html)
+* [RISC-V GNU toolchain](https://github.com/riscv/riscv-gnu-toolchain)
+* [QEMU](https://github.com/qemu/qemu)
+* [RISC-V Linux QEMU](https://risc-v-getting-started-guide.readthedocs.io/en/latest/linux-qemu.html)
+
+!!! note
+    An environment variable `RISCV_TOOLCHAIN_ROOT` needs
+    to be set to the root directory of the installed GNU toolchain. The variable
+    can be used in building the RISCV target and a LLVM AOT module.
```

Comment:
grammar nit: IMO the correct article would be "_an_ LLVM AOT module", if LLVM is pronounced "el el vee em" (which starts with a vowel sound). See also https://developers.google.com/style/articles.

To get around the grammar issue, you could reword so no article is needed:

```suggestion
!!! note
    The `RISCV_TOOLCHAIN_ROOT` environment variable needs to be set
    to the root directory of the installed GNU toolchain when building and
    using the RISCV compiler target.
```

(does it need to be set for both the build _and_ run steps, or just the run step for linker discovery?)

Reply:
It's definitely required for linker discovery, but it is also useful to set it for the target runtime compilation, as in the target configuration the CLI sets 
```shell
-DRISCV_TOOLCHAIN_ROOT=${RISCV_TOOLCHAIN_ROOT}
```

Reply:
Reword the note based on the comment.